src/ptf/packet.py

```diff
@@ -16,7 +16,7 @@
     import scapy.layers.dhcp
     import scapy.packet
     import scapy.main
-    if not config["disable_ipv6"]:
+    if "disable_ipv6" not in config:
```

Comment:
Then it should be `if not config.get("disable_ipv6", default=False)` to get today's behavior and solve your import issue.